cmd/flux/diff_kustomization.go

```diff
@@ -66,6 +68,8 @@ func init() {
 	diffKsCmd.Flags().StringVar(&diffKsArgs.kustomizationFile, "kustomization-file", "", "Path to the Flux Kustomization YAML file.")
 	diffKsCmd.Flags().BoolVar(&diffKsArgs.strictSubst, "strict-substitute", false,
 		"When enabled, the post build substitutions will fail if a var without a default value is declared in files but is missing from the input vars.")
+	diffKsCmd.Flags().BoolVarP(&diffKsArgs.recursive, "recursive", "r", false, "Recursively diff Kustomizations")
+	diffKsCmd.Flags().StringToStringVar(&diffKsArgs.localSources, "local-sources", nil, "Local sources")
```

Comment:
Please specify the format in the flag description and also add an example to the `Long` description of the command.

Reply:
@stefanprodan ok, added. And I probably should add `--recursive` to `flux build kustomization` as well.

Reply:
Yes, buid and diff must have the same functionality, also please add tests.

Reply:
@stefanprodan ok, I've added build functionality and tests.